### PR TITLE
Feat(eos_designs)!: Shutdown interfaces and bgp towards undeployed peers by default

### PR DESCRIPTION
**Pull request by Shivani-gslab**
## Change Summary

Flip default of shutdown_bgp_towards_undeployed_peers & shutdown_interfaces_towards_undeployed_peers

## Related Issue(s)

Fixes #[159](https://github.com/aristanetworks/avd-internal/issues/159)

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Flip default of shutdown_bgp_towards_undeployed_peers & shutdown_interfaces_towards_undeployed_peers

## How to test
Molecule tests

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 1]
```diff
@@ -338,6 +338,18 @@ To retain the previous behavior set the value back to `node_id`:
 +isis_system_id_format: node_id
 ```
 
+### Default value of `shutdown_bgp_towards_undeployed_peers` and `shutdown_interfaces_towards_undeployed_peers` is changed to `true`
+
+In AVD 4.0.0, the default value of `shutdown_bgp_towards_undeployed_peers` and `shutdown_interfaces_towards_undeployed_peers` was `false`.
+In AVD 5.0.0, the default value of `shutdown_bgp_towards_undeployed_peers` and `shutdown_interfaces_towards_undeployed_peers` is changed to `true`.
+
+To configure `no shutdown`, set the value to `false`:
```
Comment (laxmikantchintakindi) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
To retain the previous behaviour, set the value to `false`:
```

**Comment by laxmikantchintakindi** [comment]
We need a test where these keys are set to false.

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 2]
```diff
@@ -338,6 +338,18 @@ To retain the previous behavior set the value back to `node_id`:
 +isis_system_id_format: node_id
 ```
 
+### Default value of `shutdown_bgp_towards_undeployed_peers` and `shutdown_interfaces_towards_undeployed_peers` is changed to `true`
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
### Shutdown interfaces and BGP towards undeployed peers by default
```

File: ansible_collections/arista/avd/docs/release-notes/5.x.x.md [hunk 3]
```diff
@@ -129,6 +129,12 @@ Previously they also accepted a string, which was autoconverted to a list.
 
 See the [porting guide](../porting-guides/5.x.x.md#) for details.
 
+### Default value of `shutdown_bgp_towards_undeployed_peers` and `shutdown_interfaces_towards_undeployed_peers` is changed to `true`
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/release-notes/5.x.x.md]:
```suggestion
#### Shutdown interfaces and BGP towards undeployed peers by default
```

**Review by MaheshGSLAB — APPROVED** [review]

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**